src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -56,7 +57,7 @@ public void Start(Wallet wallet)
         {
```

Comment:
Can we add an optional Settings parameter to the Start method that overrides whatever was loaded by Configure?

Reply:
Suggestion:

```
        public void Start(Wallet wallet, Settings settings = null)
        {
            if (started) return;
            if (settings != null) this.settings = settings;
            started = true;
            consensus = neoSystem.ActorSystem.ActorOf(ConsensusService.Props(neoSystem, this.settings, wallet));
            consensus.Tell(new ConsensusService.Start());
        }
```

Reply:
Done.